nautilus_core/accounting/src/account/cash.rs

```diff
@@ -158,7 +158,7 @@ impl DerefMut for CashAccount {
 
 impl PartialEq for CashAccount {
     fn eq(&self, other: &Self) -> bool {
-        self.id == other.id
+        self.id == other.id && self.events == other.events
```

Comment:
An account, like an order etc - should only be compared with its `id`.

Reply:
Yes, I only added this becase tests we passing when i was comparing two different pyo3 CashAccount objects with different events, and `id` was only checked and it was equal, will revert this